### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by mroz22**

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ const TrezorConnect: API = {
         eventEmitter.removeListener(type, fn);
     },
 
+    removeAllListeners: () => {
+        eventEmitter.removeAllListeners();
+    },
```
Comment (mroz22) [inline src/js/index.js:34]: add types to typescript

File: tests/common.setup.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    await controller.send({ type: 'emulator-start', version: '2.2.0' });
+    // todo: find a way how to remove these sleeps
+    await sleep(501);
+    await controller.send({ type: 'bridge-start' });
+    await sleep(501);
+    if (options.wipe) {
+        await controller.send({ type: 'emulator-wipe' });
+    } else {
+        const mnemonic = typeof options.mnemonic === 'string' && options.mnemonic.indexOf(' ') > 0 ? options.mnemonic : MNEMONICS[options.mnemonic];
+        console.log(mnemonic)
```
Comment (szymonlesisz) [inline tests/common.setup.js]: rm

File: tests/common.setup.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    const onUiRequestButton = async (event) => {
+        controller.send({ type: 'emulator-decision' });
+    }
+
+    TrezorConnect.removeAllListeners();
+
+    await TrezorConnect.init({
+        // connectSrc: 'http://localhost:8099/_karma_webpack_/',
```
Comment (mroz22) [inline tests/common.setup.js]: rm

File: tests/run.sh [hunk 4]
```diff
@@ -0,0 +1,38 @@
+
+function cleanup {
+  # print out all docker logs on cleanup
+  # docker logs $id
+  # stop trezor-env container
+  docker stop $id
+  ps aux | grep python3
```
Comment (mroz22) [inline tests/run.sh]: rm

**Comment by szymonlesisz** [comment]
nice one!

**Pull request merged**